infra/app/env-template/main.tf

```
@@ -29,6 +29,7 @@ locals {
   side_load_s3_name                       = "${local.project_name}-side-load-${var.environment_name}"
   contact_email                           = "wic-projects-team@navapbc.com"
   staff_idp_client_domain                 = "${var.environment_name}-idp.wic-services.org"
+  waf_name                                = "${local.project_name}-${local.project_name}-waf"
```

🤔 Did you want `local.project_name` twice?
```suggestion
  waf_name                                = "${local.project_name}-waf"
```

---

I put the project name twice because our WAF name somehow ended up resolving to `wic-prp-wic-prp-waf` due to how the `local.project` and `local.app` in the `app-waf/main` were  defined.

---

In an ideal situation it would look like your suggestion